tidy3d/web/api/webapi.py

```diff
@@ -926,6 +927,35 @@ def real_cost(task_id: str, verbose=True) -> float:
     return flex_unit
 
 
+@wait_for_connection
+def account(verbose=True) -> Account:
+    account_info = Account.get()
+    if verbose and account_info:
+        console = get_logging_console()
+        credit = account_info.credit
+        credit_expiration = account_info.credit_expiration
+        cycle_type = account_info.allowance_cycle_type
+        cycle_amount = account_info.allowance_current_cycle_amount
+        cycle_end_date = account_info.allowance_current_cycle_end_date
+        free_simulation_counts = account_info.daily_free_simulation_counts
+
+        message = ""
+        if credit:
+            message += f"Current FlexCredit balance: {credit:.2f}"
+            if credit_expiration:
+                message += f" and expiration date: {credit_expiration}. "
+            else:
+                message += ". "
+        if cycle_type and cycle_amount and cycle_end_date:
+            message += f"{cycle_type} FlexCredit balance: {cycle_amount:.2f} and expiration date: {cycle_end_date}. "
+        if free_simulation_counts:
+            message += f"Daily free simulation counts: {free_simulation_counts}."
```

Comment:
Same comment as above, better check `free_simulation_counts is not None`. Also let's rephrase the message to "Remaining daily free simulations"